src/docfx/lib/log/CustomRule.cs

```diff
@@ -16,8 +16,22 @@ internal class CustomRule
 
         public bool CanonicalVersionOnly { get; private set; }
 
+        public bool PullRequestOnly { get; private set; }
+
         public CustomRule() { }
 
-        public CustomRule(ErrorLevel? severity) => Severity = severity;
```

Comment:
Keep this constructor to make `ShortHandConverter` work. There is an existing shorthand to turn off a rule as illustrated in the test.

Reply:
Got it